Find a functionally equivalent code:android.app.Fragment.getLayoutInflater:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide Hack so that DialogFragment can make its Dialog before creating
 * its views, and the view construction can use the dialog's context for
 * inflation.  Maybe this should become a public API. Note sure.
 */

@@ -1,3 +1,12 @@
 {
-    return mActivity.getLayoutInflater();
+    // Newer platform versions use the child fragment manager's LayoutInflaterFactory.
+    if (mActivity.getApplicationInfo().targetSdkVersion >= Build.VERSION_CODES.LOLLIPOP) {
+        LayoutInflater result = mActivity.getLayoutInflater().cloneInContext(mActivity);
+        // Init if needed; use raw implementation below.
+        getChildFragmentManager();
+        result.setPrivateFactory(mChildFragmentManager.getLayoutInflaterFactory());
+        return result;
+    } else {
+        return mActivity.getLayoutInflater();
+    }
 }

